### PR TITLE
updated dependancies

### PR DESCRIPTION
**Pull request by schuetta92**

### CODE REVIEW

File: src/authorizeStep.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ export class AuthorizeStep {
         return next.cancel(new Redirect(loginRoute));
       }
     }
-    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment) == loginRoute) {
-      var loginRedirect = this.auth.getLoginRedirect();
+    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment == loginRoute)) {
+      var loginRedirect = this.auth.getLoginRedirect().replace(/\/#/gi, '');
```
Comment (RWOverdijk) [inline src/authorizeStep.js]: Why did you add this replace?

**Comment by RWOverdijk** [comment]
Closing this one in favor of #51. Thank you for your effort though! :beers:

File: src/authorizeStep.js (hunk 1 above)
Comment (schuetta92) [inline src/authorizeStep.js]: Sorry this wasn't suppose to be in this pull request. 

But the replace needs to be there anyways since the login redirect route is in the format of "/#customers" or "/#/customers" then new Redirect() doesn't see it as a route and it doesn't work. It needs to be in the format of "customers" or "/customers" it only works on login since the redirect is using window.href  =

Comment (RWOverdijk) [inline src/authorizeStep.js]: I don't think this belongs here, personally. I think this should be changed in the configuration in stead of using a hack here.

Comment (schuetta92) [inline src/authorizeStep.js]: You're right. I just didn't want to make a breaking change. I'll make a better fix and make a different pull request. Thanks!
